SpywareFilter/sections/specific.txt

```diff
@@ -967,7 +967,7 @@ europcar.com###wt_overlay
 .arcor.de/pixel/
 .arnostat.nl/click.
 m24.ru##.b-counters > ul
-udaff.com##.counters
+vesti-ua.net,udaff.com##.counters
```

Comment:
Unsure if that left-over is correct for Spyware filter.

Reply:
why not? We hide ad leftovers in ad blocking filters.

Reply:
Yes, but this is Spyware filter.